app/src/main/assets/AIMI/variant Pam/determine-basal.js

```diff
@@ -0,0 +1,1885 @@
+/*
```

Comment:
Oops, these files are not supposed to be here right?

Reply:
woah, sorry :) pushing without it. took commit from my branch